settings.gradle.kts

```diff
@@ -63,6 +63,7 @@ dependencyResolutionManagement {
             version("io.grpc", "1.65.1")
             version("io.grpc.protobuf", "1.65.1")
             version("io.grpc.stub", "1.65.1")
+            version("io.grpc.netty.shaded", "1.65.1")
 
             // Reference from the protobuf plugin
             version("google.proto", "4.27.3")
```

Comment:
This should be updated to 4.28.2 to bring in the latest patches.

Reply:
according to `maven central` we are using almost the latest version.

wanted to avoid using the latest on purpose while they don't release a `patch` version am assuming it could be buggy.

https://mvnrepository.com/artifact/io.grpc/grpc-netty-shaded